app/models/email_template.rb

```diff
@@ -42,6 +42,10 @@ def users
       User.mentee.where.not(registration_token: nil)
     when :unregistered_mentors
       User.mentor.where.not(registration_token: nil)
+    when :abandoned_mails_mentors
+      User.mentor.where(is_missing: true).where(missing_since: ((Time.now - 10.days)..Time.now))   
```

Comment:
@DMips for that i need to add warning_emails_send_at in db in user table. Should i add ? The query will now send mails to warning_emails_send_at > 10 days and no reply.